autofocus/build_dataset/lpz_2016_2017/process_raw.py

```diff
@@ -96,7 +96,19 @@ def _delete_bad_images(run_record):
 
     return run_record
 
-
+def _extract_seasons (FileName) :
```

Comment:
Nice, thanks. My only requests are related to style. Please run `black` and `flake8` to ensure compliance with PEP 8 and repo standards (e.g. snake_case for variable names and standards for whitespace). Instead of e.g. `FileName.startswith("JA") or FileName.startswith("WI")` you can do `FileName.startswith("JA", "WI")`.